ts/node.ts

```diff
@@ -1,9 +1,44 @@
-if (typeof fetch === "undefined") {
+function getGlobal(): any | undefined {
```

Comment:
I borrowed this from `@stardazed/streams-polyfill` to ensure that we are patching the same global object that they are.